packages/ui-extensions-react/src/surfaces/checkout.ts

```diff
@@ -50,10 +50,12 @@ export type {
   MapLocation,
   MaybeConditionalStyle,
   MaybeResponsiveConditionalStyle,
+  IconSource,
```

Comment:
Those seemed to be missing. Other non-prop exports of components , like `MapBounds` are exported here as well.